daras_ai_v2/gooey_builder.py

```
@@ -225,3 +192,121 @@ def get_conversation_data_for_saved_run(
+
+    return workflow_url
+
+
+def get_default_builer_run_url() -> str:
+    try:
+        bi = BotIntegration.objects.get(id=settings.GOOEY_BUILDER_INTEGRATION_ID)
+    except BotIntegration.DoesNotExist:
+        raise fastapi.HTTPException(
+            status_code=404,
+            detail="gooey builder deployment not found",
+        )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)